syntax = "proto3";

package fixbug;

option cc_generic_services = true;

message FriendResultCode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

message GetFriendListRequest
{
    uint32 userid = 1;
}

message GetFriendListResponse
{
    FriendResultCode result = 1;
    repeated bytes friends = 2;
}

service FriendServiceRpc
{
    rpc GetFriendsList(GetFriendListRequest) returns(GetFriendListResponse);
}